src/renderer/components/NewTeamModal.tsx

```diff
@@ -115,9 +123,17 @@ export default class NewTeamModal extends React.PureComponent<Props, State> {
     }
 
     handleTeamUrlChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        this.setState({
-            teamUrl: e.target.value,
-        });
+        const teamUrl = e.target.value;
+        this.setState({teamUrl});
+        if (!teamUrl.startsWith('http') && teamUrl.length >= 4) {
+            this.addProtocolToUrlThrottled(teamUrl);
+        }
+    }
+
+    addProtocolToUrl = (teamUrl: string): void => {
```

Comment:
I think we can avoid having this displayed to the user. I understand trying to 'teach' the user how to correctly form a URL, but this isn't something we have to be responsible for. For example, most browsers don't do this.

I'm thinking instead we can, on save, ping the URLs as we're doing here and adding the `http` after the fact. It will appear in the config and when you go to edit the URL later, but I don't think adding it while the user is typing is the correct solution here.

Reply:
Agree on this one, also if the user types it without protocol, it then adds the protocol and the user goes back and changes the domain, the result might no longer be valid.

This is meant as a helper for users that are not super technical and might not know about http/s, so it should happen at the end of the process without them having to care.

Reply:
I decided to do the ping when `save()` is called. Is that okay, or is there something better?